cfme/v2v/migration_settings.py

```diff
@@ -25,28 +27,74 @@
 from widgetastic_manageiq import WaitTab
 
 
-class MigrationSettingsView(BaseLoggedInPage):
+class MigrationThrottling(Updateable, NavigatableMixin):
+
+    def __init__(self, appliance, migration_settings):
+        self.appliance = appliance
+        self.migration_settings = migration_settings
+
+    def set_max_migration_per_conv_host(self, migration_per_conv_host):
+        """Migration throttling for conversion host
+        Args:
+            migration_per_conv_host: Integer
+        """
+        view = navigate_to(self.migration_settings, 'MigrationSettings')
+        view.tabs.migration_throttling.fill({'migration_per_conv_host': migration_per_conv_host})
+        view.tabs.migration_throttling.apply_btn.click()
+
+
+class MigrationSettings(Updateable, NavigatableMixin):
+
+    def __init__(self, appliance):
+        self.appliance = appliance
+
+    @property
+    def migration_throttling(self):
+        """The 'Migration Throttling' tab """
+        return MigrationThrottling(self.appliance, self)
+
+
+class MigrationThrottlingForm(View):
+    migration_per_conv_host = Input(id='max_concurrent_tasks_per_host')
+    migration_per_provider = Input(id='max_concurrent_tasks_per_ems')
+    apply_btn = Button("Apply")
+
+
+class ConversionHostForm(View):
     configure_conversion_host = Text(locator='(//a|//button)[text()="Configure Conversion Host"]')
     conversion_host_progress = ConversionHostProgress()
 
-    @View.nested
-    class MigrationThrottlingTab(WaitTab):  # noqa
-        fill_strategy = WaitFillViewStrategy("15s")
-        TAB_NAME = 'Migration Throttling'
 
-    @View.nested
-    class ConversionHostsTab(WaitTab):  # noqa
-        fill_strategy = WaitFillViewStrategy("20s")
-        TAB_NAME = 'Conversion Hosts'
-        title = Text('.//div[contains(@class, "pull-left")]//h3')
+class MigrationSettingsView(BaseLoggedInPage):
 
-        @property
-        def is_displayed(self):
-            return self.title.text == "Configured Conversion Host"
+    @View.nested
+    class tabs(View):  # noqa
+        """The tabs on the page"""
+
+        @View.nested
+        class migration_throttling(WaitTab):  # noqa
+            fill_strategy = WaitFillViewStrategy("15s")
+            TAB_NAME = 'Migration Throttling'
+            including_entities = View.include(MigrationThrottlingForm, use_parent=True)
+
+        @View.nested
+        class conversion_hosts(WaitTab):  # noqa
+            fill_strategy = WaitFillViewStrategy("20s")
```

Comment:
How did we decide "15s" or "20s" ? Trying to understand out of curiosity.

Reply:
In this case '15s' was less in some runs so I increased it a little .

Reply:
Got it. as long as you have sufficient confident that these values are good enough. :)